reset_security_server.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Script should be run as sudo
+# Usage reset_security_server.sh -n=server(s) -h=path/to/hosts.txt -a=path/to/ansible
+# Usage example: reset_security_server.sh -n=ss3,ss4 -h=../../X-Road/ansible/hosts/lxd_hosts.txt -a=../../X-Road/ansible
+
+if [ "$#" -ne 3 ]; then
+    echo "Usage: reset_security_server.sh -n=name(s) -h=hosts.txt -a=ansible_folder"
+    exit 1
+fi
+
+for i in "$@"
+do
+case $i in
+    -n=*|--name=*)
+    NAME="${i#*=}"
+    shift
+    ;;
+    -h=*|--hosts=*)
+    HOSTS="${i#*=}"
+    shift
+    ;;
+    -a=*|--ansible=*)
+    ANSIBLE="${i#*=}"
+    shift
+    ;;
+    *)
+        echo "Usage: reset_security_server.sh -n=name(s) -h=hosts.txt -a=ansible_folder"
+        exit 1
+    ;;
+esac
+done
+
+names=$(echo "$NAME" | tr "," "\n")
+container_list=$(lxc list --format=json | jq -r '.[] | .name + ","')
+containers=$(echo "$container_list" | tr "," "\n")
+
+for name in $names
+do
+    for container in $containers
+    do
+        if [ $name = $container ]
+        then
+            printf "\n Deleting LXD container %s\n" "$name"
+            lxc delete "$name" -f
+        fi
+    done
+done
+
+cd $ANSIBLE
+printf "\nRunning Ansible script: %s/ansible-playbook -i %s xroad_init.yml\n" "$ANSIBLE" "$HOSTS"
+ansible-playbook -i "$HOSTS" xroad_init.yml
+
+printf "\n Ansible script finished installing security servers \n"
```

Comment:
This message might be better given separately for **all** security servers that were specified with ``-n`` and were successfully installed.

Reply:
The Ansible script installs as many security servers you have specified in your "lxd_hosts.txt" file, because Ansible depends on that file and installs according to the instructions given in that file. For example, if you want to re-initialize just one security server by giving it by a flag like that: "-n=ss1", but you have specified "ss1" and "ss2" in your "lxd_hosts.txt" file, Ansible wiill re-install both, the ss1 and ss2. Hence, the Ansible script is run only one, not several times for each of the security servers.

Reply:
Amended the message